compiler/rustc_data_structures/src/lib.rs

```diff
@@ -150,3 +150,14 @@ pub fn make_display(f: impl Fn(&mut fmt::Formatter<'_>) -> fmt::Result) -> impl
 // See comments in src/librustc_middle/lib.rs
 #[doc(hidden)]
 pub fn __noop_fix_for_27438() {}
```

Comment:
> __noop_fix_for_27438

what the heck is this

Reply:
good solutions for good problems (some windows shared library shenanigans apparently?)